src/vs/github1s/notification.ts

```diff
@@ -9,21 +9,30 @@ const NOTIFICATION_STORAGE_KEY = 'GITHUB1S_NOTIFICATION';
 // Change this if a new notification should be shown
 const NOTIFICATION_STORAGE_VALUE = '20210212';
 
-const notification = {
+const getGitpodRepoUrl = () => window.location.pathname.match(/^\/[^\/]+\/[^\/]+/) ? `https://gitpod.io/#https://github.com${window.location.pathname.match(/^\/[^\/]+\/[^\/]+/)![0]}` : '';
```

Comment:
On the deploy https://github1s-git-fork-mikenikles-integrate-with-gitpod-4fd92e.vercel.app/
It gives me the same URL as the current URL but not the gitpod link.

![image](https://user-images.githubusercontent.com/503123/110569706-b23d9080-8109-11eb-8b7b-ed3aa85d49ca.png)

Reply:
That's because the development environment starts without a repository in the URL. If you open https://github1s-git-fork-mikenikles-integrate-with-gitpod-4fd92e.vercel.app/conwnet/github1s, the button link is ok.

From what I can tell, github1s manually loads `conwnet/github1s` if no project is specified in the URL. Do you think it makes sense to update the URL to `/conwnet/github1s` in the development environment if no other project is specified?

Reply:
Got it, I think it's ok to leave it as it is.